neuror/main.py

```diff
@@ -279,23 +281,27 @@ def __init__(self, inputfile, axons=None, seed=0, plane=None):
         else:
             self.apical_section = None
 
-    def run(self, outputfile, plot_file=None):
+    def run(self, outputfile, plot_file=None):  # pylint: disable=too-many-locals
         '''Run'''
         if self.cut_leaves.size == 0:
             L.warning('No cut leaves. Nothing to repair for morphology %s', self.inputfile)
             self.neuron.write(outputfile)
             return
 
-        # The only purpose of the 'planes' variable is to keep
-        # each plane.morphology alive. Otherwise sections become unusable
+        # The only purpose of the 'keep_alive' variable is to keep
+        # morphologies alive. Otherwise sections become unusable
         # https://github.com/BlueBrain/MorphIO/issues/29
-        planes = []
+        keep_alive = []
```

Comment:
Are the donor sections coming from a `mut` or 'normal' MorphIO object?  The above issues was for 'normal', and isn't a problem, afaik.  The `mut` version is this bug: https://github.com/BlueBrain/MorphIO/issues/161

Reply:
Yes, you're right. This is no longer needed. It indeed uses `morphio.Morphology` so we are good here.